templates/java/libraries/okhttp-gson/api.mustache

```diff
@@ -509,9 +509,24 @@ public class {{classname}} {
 
         @Override
         public Video uploadPart(File part, boolean isLastPart, UploadPartProgressListener uploadProgressListener) throws ApiException {
-            Integer lastPart = this.part;
-            this.part++;
-            ApiResponse<Video> localVarResp = {{operationId}}PartWithHttpInfo({{#allParams}}{{^isFile}}this.{{paramName}}, {{/isFile}}{{/allParams}}part, {{#vendorExtensions.x-client-copy-from-response}}this.{{paramName}}, {{/vendorExtensions.x-client-copy-from-response}}lastPart, isLastPart, uploadProgressListener);
+            Integer lastPart = this.partId;
+            this.partId++;
+            return uploadPart(part, this.partId, isLastPart, uploadProgressListener);
```

Comment:
I think it won't work anymore because it will start with part 2. I don't really remember the reason of this lastPart variable... It looks useless... Maybe something like that would be better:
```java
{
   this.partId++;
   return uploadPart(part, this.partId - 1, isLastPart, uploadProgressListener);
```

Reply:
Ok, I did:
`return uploadPart(part, this.partId++, isLastPart, uploadProgressListener);`